src/Catalog/Dnx/DnxCatalogCollector.cs

```diff
@@ -173,6 +195,70 @@ private async Task UpdatePackageVersionIndexAsync(IEnumerable<CatalogEntry> cata
             string version,
             Uri sourceUri,
             CancellationToken cancellationToken)
+        {
+            var shouldUseSourceStorage = _sourceStorage != null;
+            var sourceBlob = await GetBlockBlobAsync(id, version) as AzureCloudBlockBlob;
```

Comment:
>GetBlockBlobAsync [](start = 35, length = 17)

What do you think of renaming `GetBlockBlobAsync` to something like `GetSourcePackageBlobAsync` to clearly indicate it gets the source blob?